src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -148,9 +154,30 @@ public Map<CreationEnum, List<?>> addUsers(List<PiUser> users, String issuerEmai
                 continue;
             }
 
+            Optional<MediaApplication> application =
+                mediaApplicationRepository.findByEmail(user.getEmail());
```

Comment:
Rather than the application repository, should this really be looking at the pi user table or the azure active directory? he user table get's cleared out each month, so I'm thinking it won't be the right source of info